charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -6,7 +6,7 @@ metadata:
   name: {{ .name }}
   namespace: {{ .root.Release.Namespace }}
   annotations:
-{{- if and .root.Values.waitForJobCompletion (or (eq .root.Values.global.actions.execute "install") (eq .root.Values.global.actions.execute "upgrade")) }}
+{{- if and (eq .root.Values.waitForJobCompletion "true") (or (eq .root.Values.global.actions.execute "install") (eq .root.Values.global.actions.execute "upgrade")) }}
```

Comment:
This change is required for helm command to wait until the install/upgrade job to be completed. Also this is required only incase of cloudk